### PR TITLE
mcafee atd - handle errors gracefully and detonate-url fix

### PR DESCRIPTION
**Pull request by bakatzir**
fixes: https://github.com/demisto/etc/issues/13566
fixes: https://github.com/demisto/etc/issues/13608

### CODE REVIEW

File: Integrations/integration-McAfee-ATD.yml [hunk 1]
```diff
@@ -586,53 +582,59 @@ script:
         return uriSuffix;
 
     def detonate(submitType ,sample, timeout, report_type, threshold, fileName):
-        sys.exit(0)
-        result = file_upload(submitType, sample, fileName)
+        result = file_upload(submitType, sample, fileName, None, None, None, None, None, None, None, None)
```
Comment (liorblob) [inline Integrations/integration-McAfee-ATD.yml]: Why send all these `None` args?

Comment (bakatzir) [reply Integrations/integration-McAfee-ATD.yml]: it uses the same uploadfile command of the basic command that has many arguments, and are not needed for the detonate command.
not sending them results a python error.

**Comment by anara123** [comment]
@bakatzir please deprecate all the detonate commands in ATD integration with `deprecated: true`

**Comment by anara123** [comment]
also there is `integration-ATD.yml` and `integration-McAfee-ATD.yml`
`integration-ATD.yml` has no fromversion, toversion, please add

**Comment by bakatzir** [comment]
@anara123  will do. there is still no full instance to work with.

**Comment by bakatzir** [comment]
Outdated, closing this PR to avoid all the conflicts and opening a new one.

**Pull request closed without merging**